public/video-ui/src/components/FormFields/TagPicker.js

```diff
@@ -34,359 +43,264 @@ export default class TagPicker extends React.Component {
-    if (e.keyCode === keyCodes.enter) {
-      const onlyWhitespace = !/\S/.test(this.state.inputString);
-      if (!onlyWhitespace) {
+    this.setState({
+      inputClearCount: this.state.inputClearCount + 1
+    });
+  }
+
+  tagsToVisible = () => {
+    this.setState({
+      tagsVisible: true
+    });
+  }
+
+  onKeyDown = (e) => {
 
-        const newInput = this.props.tagType === TagTypes.youtube ? this.getYoutubeInputValue() : this.state.inputString;
+    this.setState({
+      showTags: true
+    });
 
-        const newFieldValue = this.state.tagValue.concat([newInput]);
+    if (e.keyCode === keyCodes.down) {
+      if (this.state.selectedTagIndex === null && this.state.capiTags.length > 0) {
 
-        this.onUpdate(newFieldValue);
         this.setState({
-          inputString: '',
-          capiTags: []
+          selectedTagIndex: 0
         });
-      }
 
-    } else if (e.keyCode === keyCodes.backspace) {
-      if (this.state.inputString.length === 0) {
-        const lastInput = this.state.tagValue[this.state.tagValue.length - 1];
-
-        if (typeof lastInput === 'string') {
-          //User is trying to delete a string input
-          this.setState(
-            {
-              lastAction: UserActions.delete
-            },
-            () => {
-              this.updateInput();
-            }
-          );
+    } else {
+        if (this.state.selectedTagIndex < this.state.capiTags.length - 1) {
+          this.setState({
+            selectedTagIndex: this.state.selectedTagIndex + 1
+          });
         }
       }
-    } else {
-      this.setState({
-        lastAction: UserActions.other
-      });
     }
-  };
 
-  renderFieldValue(value, index) {
-    if (value.webTitle) {
-      return (
-        <span key={`${value.id}-${index}`}>
-          <span className="form__field__tag__display">{value.webTitle}</span>
-          {' '}
-        </span>
-      );
-    } else {
-      return `${value} `;
+    if (e.keyCode === keyCodes.up) {
+      if (this.state.selectedTagIndex && this.state.selectedTagIndex !== 0) {
+        this.setState({
+          selectedTagIndex: this.state.selectedTagIndex - 1
+          });
+      }
     }
-  }
 
-  renderTags(tag) {
-    const addTag = () => {
-      const valueWithoutStringDupes = removeStringTagDuplicates(
-        tag,
-        this.state.tagValue
-      );
-      const newFieldValue = valueWithoutStringDupes.concat([tag]);
+    if (e.keyCode === keyCodes.enter && this.state.selectedTagIndex !== null) {
+      const newTag = this.state.capiTags[this.state.selectedTagIndex];
+      const valueWithoutDupes = this.props.tagType === TagTypes.contributor ? removeStringTagDuplicates(newTag, this.state.tagValue) : removeTagDuplicates(newTag, this.state.tagValue);
+      const newFieldValue = valueWithoutDupes.concat([newTag]);
 
       this.setState({
-        inputString: ''
+        selectedTagIndex: null,
+        inputClearCount: this.state.inputClearCount + 1
```

Comment:
To send a message from the `TagPicker` to its children to clear the input string because a new tag has been selected by navigating the tag list with arrow keys. If the count changes, the children know they should set the input string to an empty string.